src/Sarif/NotYetAutoGenerated/Run.cs

```diff
@@ -102,21 +102,21 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// Automation details that describe this run.
         /// </summary>
-        [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false)]
-        public RunAutomationDetails Id { get; set; }
+        [DataMember(Name = "automationDetails", IsRequired = false, EmitDefaultValue = false)]
+        public RunAutomationDetails AutomationDetails { get; set; }
 
         /// <summary>
         /// Automation details that describe the aggregate of runs to which this run belongs.
         /// </summary>
-        [DataMember(Name = "aggregateIds", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "RunAggregates", IsRequired = false, EmitDefaultValue = false)]
```

Comment:
>RunAggregates [](start = 28, length = 13)

the ballot text didnt actually mention that we must update this specific property, however it felt like the right thing to do, considering that we are renaming id as well. I checked with Larry offline and he suggested to use the name "RunAggregates". @Mikefan, please validate if this change is ok and/or requires a different name.

Reply:
Sounds good, I updated #341 with this.

---
In reply to: [267608983](https://github.com/Microsoft/sarif-sdk/pull/1360#discussion_r267608983) [](ancestors = 267608983)